package.json

```diff
@@ -95,6 +99,10 @@
   },
   "devDependencies": {
     "@babel/core": "^7.17.7",
+    "@emotion/react": "^11.10.0",
+    "@mantine/core": "5.3.0",
+    "@mantine/form": "^5.1.1",
+    "@mantine/hooks": "^5.0.2",
```

Comment:
Shouldn't we pin all of them to `~5.3.0` then? Mixing it doesn't make too much sense I guess. And why not include patch versions?

Reply:
@dvzacharycutler I agreed with Michael that we use `~5.3.0` for all Mantine packages. Please change.

Reply:
Done